src/Semantic/Git.hs

```diff
@@ -24,22 +30,40 @@ clone url path = sh $ do
 -- | git cat-file -p
 catFile :: FilePath -> OID -> IO Text
 catFile gitDir (OID oid) = sh $ do
-  run "git" [pack ("--git-dir=" <> gitDir), "cat-file", "-p", oid]
+  run "git" ["-C", pack gitDir, "cat-file", "-p", oid]
```

Comment:
Didn't know about this! Looks like it landed a few dot-releases ago, so this is a 👍 for me.